style.css

```diff
@@ -515,3 +515,423 @@ footer {
   margin-left: auto;
   margin-right: auto;
 }
+
+#nav-bar {
+  display: none;
+}
+
+.under-img {
+  display: none;
+}
+
+.page2-header {
+  display: none;
+}
+
+.page3-header {
+  display: none;
+}
+
+.desk-about-me-paragraph {
+  display: none;
+}
+
+.sele {
+  display: none;
+}
+
+/* desktop version */
+
+@media (min-width: 730px) {
+  #nav-bar {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 151px;
+    align-items: center;
+    flex: 1;
+  }
+
+  .header-container {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    height: 72px;
+  }
+
+  .nav-link {
+    font-size: 15px;
+    color: #fff;
+    text-decoration: none;
+    margin-right: 32px;
+    text-align: center;
+    align-items: center;
+    margin-bottom: 26px;
+  }
+
+  #features {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .logo-button {
+    display: none;
+  }
+
+  .logo {
+    display: flex;
+    width: 84px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-size: 20px;
+    font-weight: 700;
+    margin-left: 154px;
+    margin-bottom: 26px;
+  }
+
+  .headline {
+    width: 100%;
+    height: 798px;
+    background-image: url(images/desk-img1.png);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: contain;
+  }
+
+  .headline-bigger {
+    width: 566px;
+    margin-left: 143px;
+    margin-top: 60px;
+    order: 1;
+  }
+
+  .headline-text {
+    width: 566px;
+    font-weight: 500;
+    font-size: 32px;
+    margin-left: 143px;
+    margin-top: 24px;
+    order: 3;
+  }
+
+  .headline-small-text {
+    width: 566px;
+    height: 123px;
+    font-weight: 500;
+    font-size: 20px;
+    margin-left: 143px;
+    margin-top: 24px;
+    order: 4;
+  }
+
+  .social-icons {
+    width: 20px;
+    height: 20px;
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    left: 0.5%;
+    right: 96.11%;
+    top: 25.37%;
+    bottom: 89.19%;
+    order: 2;
+    margin-top: 140px;
+    margin-left: 36px;
+    margin-right: 90px;
+  }
+
+  .background-img {
+    display: none;
+  }
+
+  .recent {
+    display: flex;
+  }
+
+  .recent-work {
+    width: 315px;
+    font-family: 400;
+    font-size: 40px;
```

Comment:
- [ ] Kindle note that `400` is not a `font-family` value, `font-family` should have the font name to be used on your element.